src/mlpack/tests/main_tests/decision_tree_test.cpp

```diff
@@ -151,8 +157,10 @@ TEST_CASE_METHOD(DecisionTreeTestFixture,
                  "[DecisionTreeMainTest][BindingTests]")
 {
   arma::mat inputData;
-  DatasetInfo info;
-  if (!data::Load("braziltourism.arff", inputData, info))
+  TextOptions opts;
+  opts.Categorical() = true;
```

Comment:
```suggestion
  TextOptions opts = Categorical;
```
I think you could do this if you wanted?  No need to change the code in this PR; I am just writing this in part to understand what all we can do with the (excellent) new loading API.

Reply:
yes we can, I can apply this in the upcoming PR's. But let me merge this one first.